LoadTests/SET_UP_TEST_ENTITIES.sql

```diff
@@ -1,126 +1,93 @@
+                (STARTING_ID + INDEX, STARTING_ID + INDEX, STARTING_ID + INDEX, 'test street', 'United Kingdom', 'NWN NWN', 3, '01/10/2019 12:26:44', '01/10/2019 12:26:44', 'User', 'True');
+
+                INSERT INTO "OrganisationSicCodes"
+                ("OrganisationSicCodeId", "OrganisationId", "SicCodeId", "Created", "Source")
+                VALUES
+                (STARTING_ID + INDEX, STARTING_ID + INDEX, 2200, '01/10/2019 12:26:44', 'User');
+
+                INSERT INTO "UserOrganisations"
+                ("UserId", "OrganisationId", "ConfirmAttempts", "Created", "Modified", "PINConfirmedDate")
+                VALUES
+                (STARTING_ID + INDEX, STARTING_ID + INDEX, 0, '01/10/2019 12:26:44', '01/10/2019 12:26:44', '01/10/2019 12:26:44');
+
+                -- Create test organisations that are NOT linked to test users
+                INSERT INTO "Organisations"
+                ("OrganisationId", "CompanyNumber", "OrganisationName", "SectorTypeId", "StatusId", "StatusDate", "StatusDetails", "Created", "Modified", "OptedOutFromCompaniesHouseUpdate", "EmployerReference")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, '99999' || CAST(INDEX + NUM_OF_USERS AS VARCHAR(16)), 'test_' || CAST(INDEX + NUM_OF_USERS  AS VARCHAR(16)), 1, 3, '01/10/2019 12:26:44', 'PIN Confirmed', '01/10/2019 12:26:44', '01/10/2019 12:26:44', true, 'ABCDE' || CAST(INDEX + NUM_OF_USERS AS VARCHAR(16)));
+
+                INSERT INTO "OrganisationNames"
+                ("OrganisationNameId", "OrganisationId", "Name", "Source", "Created")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, 'test_' || CAST(INDEX + NUM_OF_USERS AS VARCHAR(16)), 'User', '01/10/2019 12:26:44');
+
+                INSERT INTO "OrganisationScopes"
+                ("OrganisationScopeId", "OrganisationId", "ScopeStatusId", "ScopeStatusDate", "RegisterStatusId", "RegisterStatusDate", "SnapshotDate", "StatusId")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, 1, '01/10/2019 12:26:44', 0, '01/10/2019 12:26:44', '05/04/2017 00:00:00', 3);
+
+                INSERT INTO "OrganisationAddresses"
+                ("AddressId", "OrganisationId", "CreatedByUserId", "Address1", "Country", "PostCode", "StatusId", "StatusDate", "Created", "Source", "IsUkAddress")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, 'test street', 'United Kingdom', 'NWN NWN', 3, '01/10/2019 12:26:44', '01/10/2019 12:26:44', 'User', 'True');
+
+                INSERT INTO "OrganisationSicCodes"
+                ("OrganisationSicCodeId", "OrganisationId", "SicCodeId", "Created", "Source")
+                VALUES
+                (STARTING_ID + NUM_OF_USERS + INDEX, STARTING_ID + NUM_OF_USERS + INDEX, 2200, '01/10/2019 12:26:44', 'User');

```

Comment:
Would it be worthwhile to update some of these dates to something more recent (ie in this reporting season)? Not a big deal otherwise

Reply:
Oh yeah can do, I wasn't sure it'd matter though - these dates aren't going to be used anywhere significant, but it's probably worth making sure they're as up to date as possible just in case! 😄